projects/fmcdaq2/src/app/fmcdaq2.c

```diff
@@ -462,6 +533,15 @@ static int fmcdaq2_trasnceiver_setup(struct fmcdaq2_dev *dev,
 {
 	int status;
 
+#ifdef JESD_FSM_ON
```

Comment:
same as my other comment, i think we should use the jesd framework and not care about the old setup method

Reply:
For the time being we will keep both implementations (with and without using the FSM).